### ISSUE TITLE
iPad Pro (11 inch) ios13 displays ugly font

### ISSUE THREAD

**Issue by wnchan2000** [flutter/flutter#41604]
Is there anyone connecting your iPad Pro to debug and find the fonts all gone boldfaced? I have tried many times and get the poor displays. I have even start a new project (test7 as shown below) and get the ugly display, and the boldface one becomes even uglier.

## Steps to Reproduce

Start a new project (the increment counter) and run on a real iPad Pro.

## Logs

```
/var/folders/q0/2kd3g8dd42l2ffydw07y4l440000gn/T/_fluttter_toolVBU5J9/app.dill
0
[        ] Updating files
[  +90 ms] DevFS: Sync finished
[        ] Syncing files to device iPad (2)... (completed in 5,642ms, longer than expected)
[        ] Synced 0.9MB.
[        ] Sending to VM service: _flutter.listViews({})
[   +1 ms] Result: {type: FlutterViewList, views: [{type: FlutterView, id: _flutterView/0x108815e20, isolate: {type: @Isolate,
fixedId: true, id: isolates/3929202075474799, name: main.dart$main-3929202075474799, number: 3929202075474799}}]}
[        ] <- accept
[        ] Connected to _flutterView/0x108815e20.
[   +1 ms] 🔥  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
[        ] An Observatory debugger and profiler on iPad (2) is available at: http://127.0.0.1:1027/DH5Yn5bhBRQ=/
[        ] For a more detailed help message, press "h". To detach, press "d"; to quit, press "q".
```
```
Analyzing test7...                                                      
No issues found! (ran in 2.3s)
```
```
[✓] Flutter (Channel master, v1.10.7-pre.78, on Mac OS X 10.14.6 18G95, locale en-HK)
    • Flutter version 1.10.7-pre.78 at /Users/chanraymond/development/flutter
    • Framework revision 839fdbd2ed (8 hours ago), 2019-09-29 21:32:06 -0700
    • Engine revision 5b952f286f
    • Dart version 2.6.0 (build 2.6.0-dev.0.0 6ff8d2199d)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/chanraymond/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.0, Build version 11A420a
    • CocoaPods version 1.8.0

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.38.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.4.1

[✓] Connected device (3 available)
    • Pixel 3 XL        • 8A9Y0G17J                            • android-arm64 • Android 9 (API 28)
    • iPad (2)          • 00008027-0001504C2E85002E            • ios           • iOS 13.1
    • iPhone 11 Pro Max • 55D3FBC2-D1EB-4723-917F-B93A13B91072 • ios           • com.apple.CoreSimulator.SimRuntime.iOS-13-0
      (simulator)

• No issues found!
```

**Comment by HansMuller** [comment]
@justinmc - Can you check to see if there's a regression for iOS 13.1 here?

**Comment by justinmc** [comment]
Here's a side by side screenshot of two simulators, iPad Pro 9.7" iOS 11.3 on the left, and iPad Pro 11" iOS 13.0 on the right.  I don't see any boldness, @wnchan2000 do I need to use a physical device?

<img width="1431" alt="Screen Shot 2019-09-30 at 8 48 35 AM" src="https://user-images.githubusercontent.com/389558/65894747-20035b80-e35f-11e9-8d69-75b09d2abfae.png">

**Comment by pouledodue** [comment]
it says "run on a real iPad Pro"

**Comment by wnchan2000** [comment]
The boldfacing problem occurs in physical device only. It displays perfectly with simulator. Also, there is no boldfacing problem when displaying Chinese characters, both in physical device and simulator.

**Comment by wnchan2000** [comment]
[https://github.com/wnchan2000/misc/blob/master/IMG_1EDB242D5763-1.jpeg](url)

**Comment by wnchan2000** [comment]
![IMG_1EDB242D5763-1](https://user-images.githubusercontent.com/8317930/65931917-1f8dc200-e43e-11e9-8a0d-93231aa628b8.jpeg)

**Comment by Jasphere-Appsploration** [comment]
Same here. Screenshot attached for iOS 13.1 on iPhone X (real phone)
![ios13](https://user-images.githubusercontent.com/25500437/65932205-3c76c500-e43f-11e9-973c-c3141075afb1.PNG)

**Comment by wnchan2000** [comment]
Screenshot using iPad Pro 11" ios 13.1
![IMG_37C41A790B7B-1](https://user-images.githubusercontent.com/8317930/65934032-77302b80-e446-11e9-800a-fd0e0b7da13b.jpeg)

**Comment by Smagold** [comment]
Saw this happen on iPhone X after iOS 13 update. Possible solution: add custom font to the app (e.g. Roboto).

**Comment by justinmc** [comment]
This is probably related to https://github.com/flutter/flutter/issues/41101.  It seems something in iOS 13 has caused Flutter's font to change.

**Comment by rogiervandenberg** [comment]
I can confirm it happens on iPhones too, not just iPad. Things that were perfectly fine, are broken (totally uglyfied) since the update from iOS 12 to 13, even without installing an updated version of our app.

**Comment by justinmc** [comment]
Is this fixed with https://github.com/flutter/flutter/issues/41101?

**Comment by wnchan2000** [comment]
Fixed with #41101
